arrows/ceres/bundle_adjust.cxx

```diff
@@ -67,51 +68,54 @@ class bundle_adjust::priv
   // a map from track id to landmark parameters
-  std::unordered_map<track_id_t, std::vector<double> > landmark_params;
+  std::unordered_map< track_id_t, std::vector< double > > landmark_params;
   // a map from frame number to extrinsic parameters
-  std::unordered_map<frame_id_t, std::vector<double> > camera_params;
+  std::unordered_map< frame_id_t, std::vector< double > > camera_params;
   // vector of unique camera intrinsic parameters
-  std::vector<std::vector<double> > camera_intr_params;
-  // a map from frame number to index of unique camera intrinsics in camera_intr_params
-  std::unordered_map<frame_id_t, unsigned int> frame_to_intr_map;
+  std::vector< std::vector< double > > camera_intr_params;
+  // a map from frame number to index of unique camera intrinsics in
+  // camera_intr_params
+  std::unordered_map< frame_id_t, unsigned int > frame_to_intr_map;
 
   // ==========================================================================
   // A class to register callbacks with Ceres
   class StateCallback
     : public ::ceres::IterationCallback
   {
   public:
-    explicit StateCallback(bundle_adjust::priv* p)
-      : parent(p) {}
+    explicit StateCallback( bundle_adjust::priv* p )
+      : parent( p ) {}
 
     ::ceres::CallbackReturnType
-      operator() (const ::ceres::IterationSummary& summary)
+    operator()( const ::ceres::IterationSummary& summary )
     {
-      if (!parent || !parent->parent)
+      if( !parent || !parent->parent )
       {
         return ::ceres::SOLVER_CONTINUE;
       }
+
       bundle_adjust& ba = *parent->parent;
-      if (parent->verbose)
+      if( parent->verbose )
       {
-        if (summary.iteration == 0)
+        if( summary.iteration == 0 )
         {
-          LOG_DEBUG(ba.logger(),
-            "iter         cost  cost_change   |gradient|       "
-            "|step|  iter_time total_time");
+          LOG_DEBUG( ba.logger(),
+                     "iter         cost  cost_change   |gradient|       "
+                     "|step|  iter_time total_time" );
         }
-        LOG_DEBUG(ba.logger(),
-          std::setw(4) << summary.iteration << " "
-          << std::setw(12) << summary.cost << " "
-          << std::setw(12) << summary.cost_change << " "
-          << std::setw(12) << summary.gradient_max_norm << " "
-          << std::setw(12) << summary.step_norm << " "
-          << std::setw(10) << summary.iteration_time_in_seconds << " "
-          << std::setw(10) << summary.cumulative_time_in_seconds);
+        LOG_DEBUG( ba.logger(),
+                   std::setw( 4 ) << summary.iteration << " " <<
+                   std::setw( 12 ) << summary.cost << " " <<
+                   std::setw( 12 ) << summary.cost_change << " " <<
+                   std::setw( 12 ) << summary.gradient_max_norm << " " <<
+                   std::setw( 12 ) << summary.step_norm << " " <<
+                   std::setw(
+                     10 ) << summary.iteration_time_in_seconds << " " <<
```

Comment:
I believe that is how uncrustify re-formatted the lines; I have added an extra line break where suggested, and back-indented the lines so that we do not go over the 80 char margin

Reply:
Yes, this sort of thing is why I think of Uncrustify less as a tool to make your code pretty and more a tool to enforce a conforming style. It's often the case that multiple options may be conforming, but one is clearly superior to another, and the tool may not pick the best one. (It's also why I loathe the clang-format "my way or the highway" approach; because it often results in sub-optimal formatting.)